Plugins/NativeInput/Source/Shared/NativeInput.cpp

```diff
@@ -166,11 +187,33 @@ namespace Babylon::Plugins
     void NativeInput::Impl::PointerMove(uint32_t pointerId, int32_t x, int32_t y, DeviceType deviceType)
     {
         m_runtimeScheduler([pointerId, x, y, deviceType, this]() {
-            std::vector<int32_t>& deviceInputs{ GetOrCreateInputMap(deviceType, pointerId, { POINTER_X_INPUT_INDEX, POINTER_Y_INPUT_INDEX }) };
-            SetInputState(deviceType, pointerId, POINTER_X_INPUT_INDEX, x, deviceInputs, true);
-            SetInputState(deviceType, pointerId, POINTER_Y_INPUT_INDEX, y, deviceInputs, true);
+            std::vector<int32_t>& deviceInputs{GetOrCreateInputMap(deviceType, pointerId, {POINTER_X_INPUT_INDEX, POINTER_Y_INPUT_INDEX, POINTER_DELTA_HORIZONTAL, POINTER_DELTA_VERTICAL})};
+            int32_t deltaX = 0;
+            int32_t deltaY = 0;
 
+            if (m_getDelta)
```

Comment:
Instead of adding a flag, you could also just check to see if the `deviceInputs` map has existing values for `POINTER_X_INPUT_INDEX`/`POINTER_Y_INPUT_INDEX` right?

Reply:
Wouldn't our initial to GetOrCreateInputMap in NativeInput::Impl::DeviceInputSystem::Initialize map an initial value of zero?

Reply:
Maybe? Don't remember, but if that's what you are seeing then I guess the `m_getDelta` is your best bet. 🤷🏼‍♂️